geometry_msgs/WrenchStamped manip_wrench
---
bool success
string message
